fenix/app/src/main/java/org/mozilla/fenix/settings/logins/fragment/EditLoginFragment.kt

```diff
@@ -83,6 +85,10 @@ class EditLoginFragment : Fragment(R.layout.fragment_edit_login), MenuProvider {
                 navController = findNavController(),
                 loginsFragmentStore = loginsFragmentStore,
                 clipboardHandler = requireContext().components.clipboardHandler,
+                snackbar = FenixSnackbar.make(
```

Comment:
Making a note that if the user is editing a login and we are showing a snackbar then it should probably also contain an "undo" button.